src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,30 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (!(object instanceof Rectangle) || object == null) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) object;
+        return (length == null || rectangle.length == null)
```

Comment:
```suggestion
        return length == null
```
this should be enough